src/components/SkipLinks/__tests__/SkipLinks.test.tsx

```diff
@@ -1,41 +1,39 @@
 import React from 'react'
 
-import { render, screen } from '@testing-library/react'
+import { screen } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
-import { IntlProvider } from 'react-intl'
 
 import '@testing-library/jest-dom'
+import render from '@/test'
```

Comment:
is it the correct path (should it be imported from test of testing-lib?)

Reply:
In the widget there is a custom "render" that has been created to include <IntlProvider> directly so we don't have to add it every time we render something in tests :). It's just a wrapper for intl